graphql/src/main/java/io/jmix/graphql/datafetcher/EntityQueryDataFetcher.java

```diff
@@ -104,6 +106,13 @@
                 Types.SortOrder sortOrder = orderByPathAndOrder.getValue();
                 query.setSort(Sort.by(
                         sortOrder == Types.SortOrder.ASC ? Sort.Order.asc(path) : Sort.Order.desc(path)));
+            } else {
```

Comment:
Code below is a candidate to be an utility method. Please extract method in `EnvironmentUtils` class (later we rename it with more convenient name).
B.t.w need to  check, may be such utility already exist in Jmix classes (metadata, metadataTools e.t.c)

Reply:
I've added the public method `io.jmix.core.MetadataTools#findLastModifiedDateProperty` for searching such properties